main.go

```diff
@@ -783,8 +796,8 @@ var mainTrigger = hbot.Trigger{
 		for _, s := range sliceUniqMap(toPrint) {
 			var prefix string
 			isPublic := strings.Contains(m.To, "#")
-			// If it's not a PM, address them
-			if isPublic {
+			// If it's not a PM, address them. 
+			if isPublic && !strings.Contains(s, "#magicjudges-rules") {
```

Comment:
We don't want to address **anyone** in rules ?

Reply:
Response when sending `!104.3a` in test channel:
`DBUG[08-09|08:43:00] Figure out what 's' is                   s="104.3a. A player can concede the game at any time. A player who concedes leaves the game immediately. That player loses the game."`

`s` doesn't seem to contain the send-to channel.

Reply:
Ohh I understand...it's because you are already addressing the person in the generated message.

This functionality should also be possible in your IRC client, but happy for the bot to support this too.
I'm assuming wc is for "Wrong Channel" ?

Reply:
>I'm assuming wc is for "Wrong Channel"
Yep. I couldn't think of anything super great for it; `rules` is too close to `rule`, and `goaway` felt too mean 😢

 I know several clients have this functionality (and I use a macro in hexchat myself), but a couple people have suggested it because they often switch between clients and either don't know how to or can't configure macros on them.